lua/cfc_time/server/storage_options/utils/mysql.lua

```diff
@@ -7,22 +7,20 @@ storage.preparedQueries = {}
 -- Maximum unsigned integer for a mysql mediumint
 storage.MAX_SESSION_DURATION = 16777215
 
+local function onError( _, err, sql )
+    logger:error( err, sql )
+end
```

Comment:
New error handler for all SQL errors.
This includes Transactions, Queries, and Prepared statements generated from the functions in this file